### ISSUE TITLE
Invalid JSON is output when -p format specified

### ISSUE THREAD

**Issue by evilpacket** [draios/sysdig#570]
When I specify a certain set of fields to output the resulting json is invalid.
sysdig 0.8.0
ubuntu 15.10

If I run this command `sysdig -bj -p"%fd.name"`

I get this output

```
,
{"fd.name":""},
{"fd.name":""},
{"fd.name":""},
{"fd.name":""},
{"fd.name":""},
{"fd.name":""},
```

As you see it doesn't have the start of the array. If I run sysdig without -p output formatting I get something like

```
[{"evt.cpu":0,"evt.dir":">","evt.info":"next=0 pgft_maj=0 pgft_min=2690 vm_size=50580 vm_rss=9284 vm_swap=0 ","evt.num":9,"ev
t.outputtime":1459485707846591574,"evt.type":"switch","proc.name":"sysdig","thread.tid":13286},
```

Which properly starts the json output.

**Comment by evilpacket** [comment]
Was thinking that if the formatter output new line delimited json (nljson) instead of starting things with an array [ and ending ] Would significantly reduce the complexity for that output format as well as be really easy still for json consumers.

**Comment by henridf** [comment]
I've run into this and wished for a non-array output too. 

The change itself would be easy, the question is if/how much it would break things.

Any thoughts @ldegio ?

**Comment by markus1189** [comment]
+1 for non-array output, especially in conjunction with `jq` this would be awesome

**Comment by evilpacket** [comment]
@ldegio would this something that we could submit a pr for? I don't want to put the work in if it's not going to be something that would be considered.

**Comment by ldegio** [comment]
a PR would be welcome!

**Comment by chrahunt** [comment]
This was fixed in #720. Now we always output newline-delimited JSON objects.
